### ISSUE TITLE
Feature request: Workspace key bindings

### ISSUE THREAD

**Issue by unkindypie** [zen-browser/desktop#1263]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Feature request: Add Workspace key bindings

This is not a bug, but a very useful feature which exist in other browser (for example, Vivaldi) - to allow to bind a workspace to specific key combination which allows to switch active workspace by pressing key binding assigned to a different workspace.
You have a separate keybindings settings section for workspaces but it lacks this option
![image](https://github.com/user-attachments/assets/079435b9-4c7c-43f9-92a9-0cfe454a00ee)
 
Basically for me this is the only reason I cannot use your browser unfortunately, I believe there is huge audience of the same people like me.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.34

### What platform are you seeing the problem on?

Linux
